<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialogClass</class>
 <widget class="QDialog" name="SettingsDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>508</width>
    <height>546</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Настройки kwFinderPlugin</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>pathToHmm:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEdit_pathToHmm"/>
   </item>
   <item row="0" column="3">
    <widget class="QToolButton" name="toolButton_pathToHmm">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>pathToDict:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="lineEdit_pathToDict"/>
   </item>
   <item row="1" column="3">
    <widget class="QToolButton" name="toolButton_pathToDict">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>pathToTiedls:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEdit_pathToTiedls"/>
   </item>
   <item row="2" column="3">
    <widget class="QToolButton" name="toolButton_pathToTiedls">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="20" column="0" colspan="4">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="19" column="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="12" column="0" colspan="4">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="13" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Директория файлов для обработки:</string>
     </property>
    </widget>
   </item>
   <item row="13" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEdit_processingDir">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="13" column="3">
    <widget class="QToolButton" name="toolButton_setProcessingDir">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="14" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Директория файлов отчётов:</string>
     </property>
    </widget>
   </item>
   <item row="14" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEdit_reportsDir"/>
   </item>
   <item row="14" column="3">
    <widget class="QToolButton" name="toolButton_setReportsDir">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="17" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Максимальное количество потоков:</string>
     </property>
    </widget>
   </item>
   <item row="17" column="1" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QSpinBox" name="spinBox_maxBPThreadCount">
       <property name="minimum">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="18" column="0" colspan="4">
    <widget class="QCheckBox" name="checkBox_useExternalProcessing">
     <property name="text">
      <string>Использовать внешний поток для рассчёта</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>PruningType:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QComboBox" name="comboBox_PruningType"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>PruningPar:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QDoubleSpinBox" name="doubleSpinBox_PruningPar"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>LoopActionsType:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="2">
    <widget class="QComboBox" name="comboBox_LoopActionsType"/>
   </item>
   <item row="6" column="0">
    <widget class="QCheckBox" name="checkBox_UseLangModel">
     <property name="text">
      <string>UseLangModel</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>NGramSize:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1" colspan="2">
    <widget class="QSpinBox" name="spinBox_NGramSize"/>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>NBestType:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1" colspan="2">
    <widget class="QComboBox" name="comboBox_NBestType"/>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_12">
     <property name="text">
      <string>NBestCount:</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1" colspan="2">
    <widget class="QSpinBox" name="spinBox_NBestCount"/>
   </item>
   <item row="11" column="0">
    <widget class="QCheckBox" name="checkBox_PrintLatticeOutput">
     <property name="text">
      <string>PrintLatticeOutput</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="11" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEdit_LatticeOutputFile"/>
   </item>
   <item row="15" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEdit_LatticeOutputDir"/>
   </item>
   <item row="15" column="3">
    <widget class="QToolButton" name="toolButton_setLatticeOutputDir">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="15" column="0">
    <widget class="QCheckBox" name="checkBox_LatticeOutputDir">
     <property name="text">
      <string>Директория для сохранения латтисов:</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="label_13">
     <property name="text">
      <string>NBestCapacity:</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1" colspan="2">
    <widget class="QSpinBox" name="spinBox_NBestCapacity"/>
   </item>
   <item row="1" column="1">
    <widget class="QCheckBox" name="checkBox_useUtf8">
     <property name="text">
      <string>Utf8</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialogClass</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>536</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialogClass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>536</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_PrintLatticeOutput</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineEdit_LatticeOutputFile</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>112</x>
     <y>311</y>
    </hint>
    <hint type="destinationlabel">
     <x>467</x>
     <y>312</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_LatticeOutputDir</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineEdit_LatticeOutputDir</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>174</x>
     <y>386</y>
    </hint>
    <hint type="destinationlabel">
     <x>289</x>
     <y>393</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_LatticeOutputDir</sender>
   <signal>toggled(bool)</signal>
   <receiver>toolButton_setLatticeOutputDir</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>138</x>
     <y>392</y>
    </hint>
    <hint type="destinationlabel">
     <x>485</x>
     <y>388</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
